submissions/EnF1NeeT/HTML-CSS_popup/index.html

```diff
@@ -0,0 +1,155 @@
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/trash.svg" alt="" class="icon"> Trash</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/calendar.svg" alt="" class="icon"> Calendar</a></li>
+              </ul>
+              <input type="checkbox" id="menu-check-more" class="input-check popup-item">
+              <label for="menu-check-more" class="check-more popup-item">↓</label>
+              <div class="main-menu-more">
+                <ul class="popup-grid-more list">
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/manage_account.svg" alt="" class="icon"> Manage accounts</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/security.svg" alt="" class="icon"> Security</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/payments.svg" alt="" class="icon"> Payments</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/store.svg" alt="" class="icon"> Store</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Settings.svg" alt="" class="icon"> Settings</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/logout.svg" alt="" class="icon"> Logout</a></li>
+                </ul>
+              </div>
+            </div>
+          </li>
+          <li class="logo"><a href="./index.html" class="logo"><h1>Kottanspost</h1></a></li>
+          <li class="greeting"><h2>Hi, Kottan!</h2><img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt=""></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="posts">
+        <article class="post">
+          <div class="topzone">
+            <img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt="post-img">
+            <p>@Jonnie</p>
+          </div>
+          <div class="post-img">
+            <img src="./img/post1.jpg" class="post-img" alt="">
+          </div>
+          <div class="action-zone">
+            <button class="btn">
+              <img src="./img/favorite_FILL0_wght400_GRAD0_opsz48.png" alt="like" class="btn-img">
+              Like
+            </button>
+            <button class="btn">
+              <img src="./img/forum_FILL0_wght400_GRAD0_opsz48.png" alt="comment" class="btn-img">
+              Comment
+            </button>
+            <button class="btn">
+              <img src="./img/send_FILL0_wght400_GRAD0_opsz48.png" alt="share" class="btn-img">
+              Share
+            </button>
+          </div>
+          <div class="post-time">posted <time datetime="24.5.3222">posted 5h ago</time></div>
+        </article>
+        <article class="post">
```

Comment:
Adding hover on the whole card is confusing to users, they think the whole card is clickable, but in your case, it's not. Also, avoid putting one interactive element into another: invalid markup, the browser doesn't know click on which element should be processed.
![image](https://user-images.githubusercontent.com/16196199/185800040-97435ed1-522f-48b1-9411-30a8d7fd5dc8.png)

Reply:
Fixed in the last commit